src/test/java/gov/loc/repository/bagit/util/PathUtilsTest.java

```diff
@@ -38,22 +38,43 @@ public void testDecode(){
   
   @Test
   public void testEncode(){
-    //just carriage return
-    Path testPath = Paths.get("/foo/bar/ham\r");
-    String expectedEncoded = "/foo/bar/ham%0D";
-    String actualEncoded = PathUtils.encodeFilename(testPath);
-    assertEquals(expectedEncoded, actualEncoded);
-    
-    //just new line
-    testPath = Paths.get("/foo/bar/ham\n");
-    expectedEncoded = "/foo/bar/ham%0A";
-    actualEncoded = PathUtils.encodeFilename(testPath);
-    assertEquals(expectedEncoded, actualEncoded);
-    
-    //both carriage return and new line
-    testPath = Paths.get("/foo/bar/ham\r\n");
-    expectedEncoded = "/foo/bar/ham%0D%0A";
-    actualEncoded = PathUtils.encodeFilename(testPath);
-    assertEquals(expectedEncoded, actualEncoded);
+	if (System.getProperty("os.name").contains("Windows")){
+	    //just carriage return
+	    String testPath = "\\foo\\bar\\ham\r";
```

Comment:
Yeah, I'm mostly thinking that we don't have a good CI story for Windows yet and would like to avoid the possibility of a future change breaking something for awhile before someone notices.